src/services/coreBackService.ts

```diff
@@ -104,8 +144,9 @@ export const postCriCallback = (
 
 export const getProvenIdentityUserDetails = (
   req: Request,
-): Promise<AxiosResponse> => {
-  return axiosInstance.get(
+): Promise<AxiosResponse<ProvenUserIdentityDetails>> => {
+  // prettier-ignore
+  return axiosInstance.get( // NOSONAR
```

Comment:
Can we accept it as a false positive in sonar instead - otherwise we'll suppress other (possibly genuine) issues on this line?

You'll need to log into Sonar with your github account, if you don't have permissions then let me know and I should be able to...

Reply:
I don't see anywhere where I can mark this as a false positive so I think I may not have permissions. Are you able to do it instead?